project/Dependencies.scala

```diff
@@ -82,10 +82,10 @@ object Dependencies {
   val mysqlConnector: ModuleID =  "mysql"                         % "mysql-connector-java"  % "8.0.30"
   val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "4.17.2"
 
-  val workbenchLibsHash = "46d1df6"
+  val workbenchLibsHash = "3de88b23-SNAP"
```

Comment:
this will get updated to the new published version once the workbench-libs PR merges